src/main/java/gov/nasa/jpl/mbee/mdk/util/GeneratorUtils.java

```diff
@@ -312,18 +312,18 @@ public static void docMetadata(Document doc, Element start) {
     }
 
     public static Stereotype getViewpointStereotype(Project project) {
-        Profile profile = StereotypesHelper.getProfile(project, SysMLConstants.SYSML_PROFILE);
+        Profile profile = StereotypesHelper.getAllProfiles(project).stream().filter
+                (x -> (MDKConstants.SYSML_PROFILE_LOCAL_ID).equals(x.getLocalID()))
```

Comment:
MDKConstants.SYSML_PROFILE_LOCAL_ID doesn't need parantheses.

Reply:
Use Converters.getElementToIdConverter.convert(x) so we can swap out the conversion without having to refactor.